fleet-be/src/main/java/com/everis/training/fleet/business/fleet/boundary/CustomerResource.java

```diff
@@ -33,27 +33,27 @@ public Response listCustomers() {
     @POST
     public Response addCustomer(Customer customer){
         con.add(customer);
-        return Response.ok().entity(String.format("The customer has been added successfully!")).build();
+        return Response.ok().entity("The customer has been added successfully!").build();
     }
 
     @Path("reserve")
     @PUT
     public Response reserveCar(Reserve reserve){
```

Comment:
A more accurate path to accomplish the REST definition should be like:  {customerId}/reserve-vehicle/{vin}